src/controllers/product/itemController.js

```diff
@@ -0,0 +1,45 @@
+import { product } from "../../database/models";
+
+const getAllItems = async (req, res) => {
+  const { page = 1, page_size = 20 } = req.query;
+
+  try {
+    const items = await product.findAndCountAll({
```

Comment:
Hello Djanati, is this endpoint meant for retrieving all items in vendor's collection*? If so, I think it would be necessary to retrieve only products by a particular vendor by checking the vendor_id on a product.